src/main/java/carracing/model/ui/InputDto.java

```diff
@@ -0,0 +1,54 @@
+package carracing.model.ui;
+
+import carracing.exception.NameNullPointerException;
+import carracing.exception.TryCountMinimumException;
+import carracing.exception.TryCountNullPointerException;
+
+public class InputDto {
+
+    private static final Integer MINIMUM_TRY_COUNT = 1;
+
+    private final String name;
+
+    private final Integer tryCount;
```

Comment:
```suggestion
    private final String name;
    private final Integer tryCount;
```
언제 한줄을 띄어쓰면 좋을지 고민되면 스타일 가이드를 한번 참고해보는 것도 좋아요 :)

https://google.github.io/styleguide/javaguide.html#s4.6.1-vertical-whitespace

Reply:
앗 진짜 상수와 변수들을 어떻게 띄어 쓰나..하고 다른 사람들 코드도 힐끔힐끔 봤는데
다 달라서 물어보려다 깜빡! 하고 있었는데 이렇게.. 어떻게.. 제 속마음을 읽으시고 .. 
감사합니다!! 😁😁